ee/ucp/admin/configure/create-audit-logs.md

```diff
@@ -4,64 +4,115 @@ description: Learn how to create audit logs of all activity in UCP
+    calls). We do not include names of created or updated resources
     - License Key
     - Remote Address
-- Request: includes all fields from the Metadata level as well as the request payload.
+
+- **Request**: includes all fields from the Metadata level as well as the 
+request payload.
+
+> Once UCP audit logging has been enabled, audit logs can be found within the 
+> container logs of the `ucp-controller` container on each UCP manager node. 
+> Please ensure you have a 
+> [logging driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set as audit logging can start to 
+> generate a lot of data. 
 
 ## Benefits
 
 You can use audit logs to help with the following use cases:
 
-- **Historical Troubleshooting** - Audit logs are helpful in determining a sequence of past events that explain why an issue occured.
-- **Security Analysis and Auditing** - Security is one of the primary uses for audit logs. A full record of all user interactions with the container infrastructure gives your security team full visibility to questionable or attempted unauthorized accesses.
-- **Chargeback** - You can use audit logs and information about the resources to generate chargeback information.
-- **Alerting** - If there is a watch on an event stream or a notification created by the event, alerting features can be built on top of event tools that generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+- **Historical Troubleshooting** - Audit logs are helpful in determining a 
+sequence of past events that explain why an issue occured.
+
+- **Security Analysis and Auditing** - Security is one of the primary uses for 
+audit logs. A full record of all user interactions with the container 
+infrastructure gives your security team full visibility to questionable or 
+attempted unauthorized accesses.
+
+- **Chargeback** - You can use audit logs and information about the resources to 
+generate chargeback information.
+
+- **Alerting** - If there is a watch on an event stream or a notification 
+created by the event, alerting features can be built on top of event tools that 
+generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+
+## Enablig UCP Audit Logging
+
+UCP audit logging can be enabled via the UCP web user interface or via the
+UCP configuration file.
+
+### Enabling UCP Audit Logging via UI
+
+1) Log in to the **UCP** Web User Interface
+
+2) Navigate to **Admin Settings**
 
-## Procedure
+3) Select **Audit Logs**
 
-1. Download the UCP Client bundle [Download client bundle from the command line](https://success.docker.com/article/download-client-bundle-from-the-cli).
+4) In the **Configure Audit Log Level** section, select the relevant logging
+level. 
 
-2.  Retrieve JSON for current audit log configuration.
+![Enabling Audit Logging in UCP](../../images/auditlogging.png){: .with-border}
 
-    ```
```

Comment:
We should still leave this in actually as an alternative to doing this over the CLI